pyproject.toml

```diff
@@ -108,12 +108,15 @@ redis = ["redis"]
 selenium = ["selenium"]
 
 [tool.poetry.group.dev.dependencies]
+black = "24.2.0"
+mypy = "1.7.1"
+pre-commit = "3.6.2"
+pg8000 = "*"
 pytest = "7.4.3"
 pytest-cov = "4.1.0"
+ruff = "0.3.0"
 sphinx = "^7.2.6"
-pg8000 = "*"
```

Comment:
Pretty sure this has to be here

Reply:
It's still there, see above. I just sorted this dependency section alphabetically.